packages/design-system-react-native/tsconfig.build.json

```diff
@@ -13,5 +13,6 @@
   "references": [
     { "path": "../design-system-twrnc-preset/tsconfig.build.json" }
   ],
-  "include": ["./src", "global.d.ts"]
+  "include": ["./src", "global.d.ts"],
+  "exclude": ["**/*.figma.tsx"]
```

Comment:
Excluding figma code connect files from build so we don't include them in the dist folder. No sure if there could be benefit to this though with figma MCP